.circleci/config.yml

```diff
@@ -0,0 +1 @@
+version: 2
```

Comment:
And why are we switching to CircleCI in this PR (and at all)? This should happen in a separate one, as this one should only be concerned with eslint.